File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main()
{
    int coins = 0;
    
    // Ask for owed change
    float change;
    do 
    {
        change = get_float("Change owed: ");
    }
    while (change <= 0);


    // Convert to cents
    int cents = round(change * 100);
    
    
    // Loop conditions for quarters 25¢, dimes 10¢, nickels 5¢ and pennies 1¢
    while (cents >= 25)
    { 
        cents = cents - 25;
        coins++;
    }
    
    while (cents >= 10)
    { 
        cents = cents - 10;
        coins++;
    }
    
    while (cents >= 5)
    { 
        cents = cents - 5;
        coins++;
    }
    
    while (cents >= 1)
    { 
        cents = cents - 1;
        coins++;
    }

    
    // Print number of coins used
    printf("%i coins\n", coins);
    
}

File: pset6/cash.py
#DONE

#ask for owed change
change = 0
while change <= 0:
    try:
        change = float(input("Change owed: "))
    except ValueError:
        pass

# Convert to cents
cents = round(change * 100)

#Loop conditions for quarters 25¢, dimes 10¢, nickels 5¢ and pennies 1¢
coins = 0
quarters = 25
dimes = 10
nickels = 5
pennies = 1

while cents >= quarters:
    cents -= quarters
    coins += 1

while cents >= dimes:
    cents -= dimes
    coins += 1

while cents >= nickels:
    cents -= nickels
    coins += 1
    
while cents >= pennies:
    cents -= pennies
    coins += 1

#print number of coins used
print(coins)




